client/redux/PendingOrders/pendingOrders.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-underscore-dangle */
+import axios from 'axios';
+import ShippedEmail from '../../components/Payment/ShippedEmail';
+
+const PENDING_ORDERS = 'PENDING_ORDERS';
+const UPDATE_SHIPPING = 'UPDATE_SHIPPING';
+
+const _fetchPendingOrders = (orders) => ({
+  type: PENDING_ORDERS,
+  orders,
+});
+
+const fetchPendingOrders = () => async (dispatch) => {
+  try {
+    const { data } = await axios.get('/api/orders/admin/pending');
+    dispatch(_fetchPendingOrders(data));
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+const _updateShipping = (shippedOrder) => ({
+  type: UPDATE_SHIPPING,
+  shippedOrder,
+});
+
+const updateShipping = (orderId) => async (dispatch) => {
+  try {
+    const { data } = await axios.put(`/api/orders/admin/pending/${orderId}`);
+    const emailText = ShippedEmail(data);
```

Comment:
Don't capitalize functions that dont return an instance of a class, or are themselves a constructor.